### PR TITLE
Fix `reactions-avatars` conflict with new style

### PR DESCRIPTION
**Pull request by fregante**
- Fix for #4674 

## Test URLs

Regular comment: https://github.com/sindresorhus/refined-github/issues/4674
Review: https://github.com/sindresorhus/refined-github/pull/4667#pullrequestreview-730139417
Review comments: https://github.com/sindresorhus/refined-github/pull/4667#discussion_r689119705

## Native

Screen Shot 10

## Refined GitHub before this PR

Screen Shot 11

## Refined GitHub in this PR

Screen Shot 9

## On white

Screen Shot 6

## While loading

Screen Shot 7

### CODE REVIEW

File: source/features/reactions-avatars.css [hunk 1]
```diff
@@ -1,44 +1,33 @@
...
-	border-bottom: 1px solid var(--github-border-color); /* Required when the first line is longer than the second line */
-	margin-bottom: -1px; /* Makes up for `order-bottom` */
-}
-
-.comment-reactions .js-pick-reaction {
-	margin-top: -1px; /* Makes up for `.reaction-summary-item {border-top}` */
-}
 
-.reaction-summary-item:not(.js-reaction-group-button) span {
+:root:root .rgh-reactions-avatar {
```
Comment (fregante) [inline source/features/reactions-avatars.css:2]: Reaction test, please react on this review comment

**Comment by ShadowJonathan** [comment]
Maybe the same padding between the number and the avatars as the number and the emoji? It looks a bit too close

**Comment by fregante** [comment]
Good suggestion. Also updated the screenshot in the readme (one of the oldest ones!)

34438653-f66535a4-ecda-11e7-9406-2e1258050cfa

**Comment by fregante** [comment]
Funny to still see the bug I just fixed locally 😩 😂 

Screen Shot 20

**Pull request merged**